scripts/hosts-add.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Add new IP-host pair to /etc/hosts.
+
+if [[ "$1" && "$2" ]]
+then
+    IP=$1
+    HOSTNAME=$2
+
+    if [ -n "$(grep $HOSTNAME /etc/hosts)" ]
+        then
+            echo "$HOSTNAME already exists:";
+            echo $(grep $HOSTNAME /etc/hosts);
+        else
+            sudo sed -i "/#### HOMESTEAD-SITES-BEGIN/c\#### HOMESTEAD-SITES-BEGIN\\n$IP\t$HOSTNAME\twww.$HOSTNAME" /etc/hosts
+
+            if ! [ -n "$(grep $HOSTNAME /etc/hosts)" ]
+                then
+                    echo "Failed to Add $HOSTNAME, Try again!";
+            fi
+    fi
+else
+    echo "Error: missing required parameters."
```

Comment:
can these all go in one `echo` statement?

Reply:
They could, but I believe the original author (whomever it was) did this so as to avoid having to insert newlines. Using `echo` in this way has the same effect as using a single call with newlines in each place that a linebreak is desired.

Reply:
ahh, good to know.

Reply:
Indeed. This is probably most legible as-is, so I'll mark this Resolved.